src/VisualStudio/Core/Impl/CodeModel/AbstractCodeModelService.cs

```diff
@@ -686,10 +686,12 @@ protected bool TryGetElementFromSource(CodeModelState state, Project project, IT
             return false;
         }
 
+        public abstract bool IsAutoProperty(SyntaxNode node);
```

Comment:
These are not called autoproperties, because the explicity do NOT declare storage for the property.